src/aggregation/aggregator_wrapper.py

```diff
@@ -0,0 +1,19 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+class VideoCLIPAggregator(nn.Module):
+    """
+    Class to wrap aggregators so we can control their normalization and logit scale
+    """
+    def __init__(self, aggregator, logit_scale=100.0, normalize=True):
+        super().__init__()
+        self.aggregator = aggregator
+        self.logit_scale = logit_scale
+        self.normalize = normalize
+
+    def forward(self, x):
```

Comment:
instead, add boolean parameters for ```prenorm``` and ```postnorm``` . If prenorm true than normalize x before the aggregator, if postnorm true normalize after the aggregator (both can be true)